src/components/HomeLeaderboardPositionCard/dummyData.ts

```diff
@@ -0,0 +1,44 @@
+const topLeaderboardList = [
+  {
+    id: 1,
+    name: 'Carlos Díaz',
+    points: 1100,
+  },
+  {
+    id: 2,
+    name: 'Jorge Delgado',
+    points: 1280,
+  },
+  {
+    id: 3,
+    name: 'Karla Sánchez',
+    points: 1100,
+  },
+];
+
+const yourPosition = {
```

Comment:
I think that at some point we should discuss how they view on the Leaderboard page will be if you are on the podium. I don't have it that clear.
You only display the podium information, and if you are third, one below (as a way to know how close the one below is from you) ?

Reply:
The podium information is only for the Teacher view. Students do get to see their position even if they are not in the podium